package.json

```diff
@@ -111,28 +111,15 @@
       "eslint-plugin-react": "^7.37.5",
       "eslint-plugin-react-hooks": "^5.2.0"
     },
-    "@leafygreen-ui/emotion": "^4.0.9",
-    "@leafygreen-ui/lib": "^15.3.0",
-    "@leafygreen-ui/palette": "^4.1.3",
-    "@leafygreen-ui/tokens": "^3.2.4",
-    "@leafygreen-ui/leafygreen-provider": "^4.0.2",
-    "@leafygreen-ui/marketing-modal": "^5.0.2",
-    "@leafygreen-ui/typography": "^20.0.2",
-    "@leafygreen-ui/icon": "^14.6.0",
-    "@leafygreen-ui/inline-definition": "^9.0.5",
-    "@leafygreen-ui/popover": "^13.0.11",
-    "@leafygreen-ui/badge": "^9.0.2",
-    "@leafygreen-ui/banner": "^10.1.0",
-    "@leafygreen-ui/button": "^22.0.2",
-    "@leafygreen-ui/hooks": "^8.3.4",
-    "@leafygreen-ui/icon-button": "^16.0.2",
-    "@leafygreen-ui/input-option": "^3.0.4",
-    "@leafygreen-ui/polymorphic": "^2.0.5",
-    "@leafygreen-ui/search-input": "^5.0.2",
-    "@leafygreen-ui/code": "^20.1.1",
-    "@leafygreen-ui/text-area": "^10.0.2",
-    "@leafygreen-ui/card": "^12.0.2",
-    "@leafygreen-ui/logo": "^10.0.2",
-    "@leafygreen-ui/tabs": "^17.0.4"
```

Comment:
Just curious what logic you used to decide on which overrides to remove and which to keep for this step of the LG update: I see that some of the ones that are left do overrides, ~but some are not~, so not obvious at a glance what's the deciding factor here

EDIT: ah, scratch that, didn't noticed it at first doing `npm ls`, seems all of them are related to the packages not being bumped in this PR specifically? Is that about right?